src/main/resources/sql/maxwell_schema.sql

```diff
@@ -1,34 +1,58 @@
-CREATE DATABASE IF NOT EXISTS `maxwell`;
+SET @c = CONCAT('CREATE DATABASE IF NOT EXISTS ', @maxwell_db)
 
-CREATE TABLE IF NOT EXISTS `maxwell`.`schemas` (
+PREPARE maxwell_stmt from @c
+
+EXECUTE maxwell_stmt
+
+DEALLOCATE PREPARE maxwell_stmt
+
+SET @c = CONCAT('CREATE TABLE IF NOT EXISTS `', @maxwell_db, '`.`schemas` (
```

Comment:
I am having trouble finding away to avoid the following
```
String sql = "CREATE DATABASE IF NOT EXISTS " + dbName;
conn.createStatement.execute(sql)
```

One option would be to put the onus on the user to ensure the database has been created.